opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -54,6 +54,22 @@
 
 _DEFAULT_OTEL_ATTRIBUTE_COUNT_LIMIT = 128
 _ENV_VALUE_UNSET = ""
+_EXCLUDED_ATTRIBUTES = {
+    # pseudo-private log-record attributes, they just get dropped
+    "args",
+    "msg",
+    "message",
+    "stack_info",
+    "exc_info",
+    "exc_text",
```

Comment:
I guess we should still discard the attributes like `taskName`, `asctime` etc.. that don't have the mapping in sematic conventions. Not that they are not useful but we only emit the attributes that sem convs defined in SDK

Reply:
I don't have an overly strong opinion on this, but sway towards emit all, or at the very least some of them. 

I can't find the source right now, but I think I've read that OTEL implementations should prioritize being written idiomatic within the language the implementation is written in over sticking to details of the general OTEL guidelines. I'd consider log-record attributes to fall into this category. As a python developer, I expect attributes like lineno, module, and funcName to exist on a logrecord.

Also, custom attributes already do get emitted.

Reply:
I don't think language idiomatic implementation suggestions should interpreted to associate with the attributes. This is an independent area. We are emitting some of them already by mapping to semconv attributes. And will emit the rest too but that will be after there is some spec guidelines about such cases.

Reply:
@a-recknagel 

Let's leave the attribute changes to a different pr if we ever decide to change them/the semantic convention guidelines change.

Reply:
ok, I'll restore the omission-list to its former length. 

Let me note though that, in my opinion, this reduces the value of what's remaining from my patch significantly. To re-iterate, what I'm fixing is that standard python log-formatting can be used to control how logrecords get transformed (where conceptually, the transformer boils down to a to_string method) before being emitted. Pseudo-code examples using the [default-formatter](https://github.com/python/cpython/blob/1a84bdc2371ada60c01c72493caba62c9860007b/Lib/logging/__init__.py#L546) and an otel-to-stdout receiver:
```
import logging
logging.info("foo")
# before
{
    "body": "foo",
    "severity_number": "<SeverityNumber.INFO: 9>",
    "severity_text": "INFO",
    [...]
    "span_id": "0x0000000000000000",
    "trace_flags": 0,
}
# after
{
    "body": "INFO:root:foo",
    "severity_number": "<SeverityNumber.INFO: 9>",
    "severity_text": "INFO",
    [...]
    "span_id": "0x0000000000000000",
    "trace_flags": 0,
}
```
As can be seen, the change isn't all that useful if no custom formatter is used. If you do want to use one, all googling will lead you to the [standard logrecord attributes](https://docs.python.org/3/library/logging.html#logrecord-attributes) as a reference, and no easy or straight-forward way to figure out why those wouldn't work with otel-log-handlers. I'll leave this note up for two days or so, and if there are no further comments here I'll treat this matter as acknowledged and will apply said change.

I hope this doesn't come off as passive aggressive, this is not my intention. It's all the same to me as I know how to use a dedicated otel-formatter or manually add attributes to a record.